### PR TITLE
`MigrateController::addDefaultPrimaryKey()` skip if 'id' already set

### PR DESCRIPTION
**Pull request by WinterSilence**
`yii migrate/create app\migrations\CreateDemoTable --fields="id:integer(11):unsigned:notNull,name:string(50)`

generate array with 2 'id' keys.

| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | #18844

### CODE REVIEW

**Comment by WinterSilence** [comment]
@samdark test added

File: framework/console/controllers/MigrateController.php [hunk 1]
```diff
@@ -606,7 +606,7 @@ protected function splitFieldIntoChunks($field)
     protected function addDefaultPrimaryKey(&$fields)
     {
         foreach ($fields as $field) {
-            if (false !== strripos($field['decorators'], 'primarykey()')) {
+            if ($field['property'] === 'id' || false !== stripos($field['decorators'], 'primarykey()')) {
```
Comment (samdark) [inline framework/console/controllers/MigrateController.php:609]: What is the reason for changing `strripos` to `stripos`?

**Review by samdark — APPROVED** [review]

**Review by Arhell — APPROVED** [review]

**Comment by samdark** [comment]
@WinterSilence would you please add a line for CHANGELOG?

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thanks.

**Pull request merged**